### PR TITLE
hide app ratings on 'new' page (bug 1126921)

### PR DESCRIPTION
**Pull request by spasovski**
Looks a bit awkward but here it is.

### CODE REVIEW

**Comment by ngokevin** [comment]
@pwalm is this intentional?

**Comment by pwalm** [comment]
@ngokevin yup, because the majority of new apps will have 0 star ratings and we don't want that to skew users first impressions.

File: src/media/css/tiles.styl [hunk 1]
```diff
@@ -146,6 +146,10 @@ $greater-than-mobile = unquote('(min-width: 330px)');
     &.unrated, &.unrated a {
         color: $earl-gray;
     }
+    // Hide rating on the "new" page. Most apps there have no rating.
+    [data-page-type~="new"] & {
```
Comment (ngokevin) [inline src/media/css/tiles.styl]: let's just define ```[data-page-type~="new"] .mkt-tile .rating``` below. Easier to read, easier to find.

File: src/media/css/tiles.styl [hunk 2]
```diff
@@ -146,6 +146,10 @@ $greater-than-mobile = unquote('(min-width: 330px)');
...
+        display: none;
+    }
 }
 
 .listing .mkt-tile {
```
Comment (ngokevin) [inline src/media/css/tiles.styl]: You can get rid if ```.listing``` I think

File: src/media/css/tiles.styl (hunk 1 above)
Comment (ngokevin) [inline src/media/css/tiles.styl]: It also might fit better in app-list.styl since it's more of a page definition

**Comment by ngokevin** [comment]
r+wc

**Comment by ngokevin** [comment]
thanks for the clarification @pwalm

**Pull request merged**